examples/fibonacci.masm

```diff
@@ -2,6 +2,8 @@
 
 begin
     repeat.1000
-        swap dup.1 add
+        swap 
```

Comment:
Changed that for better readability. Almost no one understands the example looking at it for the first time